File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    temp = s.split(' ')
    for i in temp:
        for j in range(len(i)):
            if j % 2 == 0:
                answer+= i[j].upper()
            if j % 2 != 0:
                answer += i[j].lower()
        answer+=" "
    
    return answer[0:-1]

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split("");
        int counter = 0;
        for(int i=0; i< str.length; i++){
            if(str[i].equals(" ")){
                counter = 1;
            }
            else if(counter % 2 == 0){
                str[i] = str[i].toUpperCase();
            }
            else if(counter % 2 != 0){
                str[i] = str[i].toLowerCase();
            }
            counter++;
            answer += str[i];
        }
        return answer;
    }
}